tsl/test/sql/continuous_aggs.sql

```diff
@@ -930,3 +930,90 @@ GROUP BY
 CALL refresh_continuous_aggregate('mat_types',NULL,NULL);
 SELECT * FROM mat_types;
 
+-------------------------------------------------------------------------------------
+-- Test issue #2616 where cagg view contains an experssion with several aggregates in
+
+CREATE TABLE water_consumption
+(
+    sensor_id   integer      NOT NULL,
+    timestamp   timestamp(0) NOT NULL,
+    water_index integer
+);
+
+SELECT create_hypertable('water_consumption', 'timestamp', 'sensor_id', 2);
+
+INSERT INTO public.water_consumption (sensor_id, timestamp, water_index) VALUES (1, '2010-11-03 09:42:30', 1030);
```

Comment:
Probably cleaner and faster to do a single statement:

```
INSERT INTO public.water_consumption (sensor_id, timestamp, water_index) VALUES 
(1, '2010-11-03 09:42:30', 1030),
(1, '2010-11-03 09:42:50', 1035),
...
```

Reply:
Originally I copied from the issue. Fixed according the suggestion.